helm/ako/templates/configmap.yaml

```diff
@@ -16,6 +16,7 @@ data:
   enableEVH: {{ .Values.AKOSettings.enableEVH | quote }}
   layer7Only: {{ .Values.AKOSettings.layer7Only | quote }}
   istioEnabled: {{ .Values.AKOSettings.istioEnabled | quote }}
+  vipPerNamespace: {{ .Values.AKOSettings.vipPerNamespace | quote }}
```

Comment:
Do we need this change in AKO-operator context?

Reply:
Thanks, had missed that, added now.